types/components/buttons.d.ts

```diff
@@ -213,6 +213,11 @@ export interface PayPalButtonsComponentOptions {
         data: Record<string, unknown>,
         actions: CreateSubscriptionActions,
     ) => Promise<string>;
+    /**
+     * Save payment methods to charge payers after a set amount of time. For example, you can offer a free trial and charge payers after the trial expires. Payers don't need to be present when charged. No checkout required.a
```

Comment:
Looks like a trailing `a`. Or maybe you meant `eh?` if you really want a [canadianism] (https://www.cs.cmu.edu/~steffan/canadianisms.html) at the end of this awesome comment.
```suggestion
     * Save payment methods to charge payers after a set amount of time. For example, you can offer a free trial and charge payers after the trial expires. Payers don't need to be present when charged. No checkout required.
```